### ISSUE TITLE
python3.6 does not work: TypeError: a bytes-like object is required, not 'str'

### ISSUE THREAD

**Issue by toabctl** [fmoo/python-editor#14]
Running the tests with python3.6 results in:
```
$ python3.6 test.py 
Traceback (most recent call last):
  File "test.py", line 5, in <module>
    use_tty='use_tty' in sys.argv)
  File "/home/tom/devel/python-editor/editor.py", line 89, in edit
    f.write(contents)
TypeError: a bytes-like object is required, not 'str'
```

**Comment by DavidRabel-b1** [comment]
#18 seems to party fix this. could you merge it, please?

**Comment by fmoo** [comment]
Fixed in 410d271631eaf49a70917522e32d1ee31e49c796
